mysensors/__init__.py

```diff
@@ -39,6 +38,7 @@ def __init__(self, event_callback=None, protocol_version="1.4"):
         self.protocol_version = protocol_version
         self.sensors = {}
         self.tasks = None
+        self.use_ack_when_set_values = kwargs.get("use_ack_when_set_values", False)
```

Comment:
I think this option belongs in `set_child_values` per message, and not a new gateway attribute. We already supported it before per message.

Reply:
@MartinHjelmare this is a workaround for smart sleep nodes.
The gateway's `set_child_values` method still allows to configure the `ACK` parameter, but for smart sleep nodes it doesn't work, because we convert set message into the new state value (which has the `ChildSensor` type and doesn't allow to store message parameters).

E.g. right now Home Assistant integration always requests echo messages during `set_child_values` calls, but for smart sleep nodes it does nothing.

Other approach would be to use a specialized type to store the new state, which would allow to save message parameters in addition to the value.
Or maybe allow to configure sensor to always use `ACK` for some types of messages.

Reply:
We could consider storing all or latest message instead of the state and replaying the message(s).

Reply:
@MartinHjelmare yes, I had the same idea, but it requires more changes. Plus storing the desired state instead of just a message is helpful when you need to reply to a state request from the node (`handle_req` function).

Reply:
@MartinHjelmare I think I've addressed all your comments except this one. We can revert this change, if you want, and leave that fix for the next PR.

Reply:
Yeah, let's remove it for now.

Reply:
@MartinHjelmare done